### PR TITLE
Jackie/styling

### PR DESCRIPTION
**Pull request by jackiewhitworth**
*Added elevation to "Running Nodes" and "Deployments" headings on the home page

### CODE REVIEW

File: src/components/ClusterOverview/ClusterOverview.js [hunk 1]
```diff
@@ -32,17 +32,18 @@ const ClusterOverview = () => {
...
     setClusterTotalPods(currentTotalPods);
   };
  
   // fetch data, then fetch again in 30 seconds
-  useEffect(async() => {
+  // when component unmounts, cancel setInterval in a cleanup function
+  useEffect(() => {
     fetchToStore();
-    setInterval(() => fetchToStore(), 30000);
+    const interval = setInterval(() => fetchToStore(), 3000)
```
Comment (ntgv93) [inline src/components/ClusterOverview/ClusterOverview.js:45]: Firing a request every 3 seconds can be quite expensive. I think 30 seconds is good enough.

File: src/components/NodeOverview/NodeOverview.js [hunk 2]
```diff
@@ -56,11 +56,13 @@ const NodeOverview = (props) => {
...
  
   // fetch data, then fetch again in 30 seconds
+  // when component unmounts, cancel setInterval in a cleanup function
   useEffect(async () => {
     fetchDataToStore();
-    setInterval(() => {
+    const interval = setInterval(() => {
       fetchDataToStore();
-    }, 30000);
+    }, 3000);
```
Comment (ntgv93) [inline src/components/NodeOverview/NodeOverview.js:64]: Firing a request every 3 seconds can be quite expensive here also.

File: src/components/K8sContainersOverview/K8sContainersOverview.js [hunk 3]
```diff
@@ -150,6 +149,14 @@ const K8sContainersOverview = (props) => {
       setMemorySaturationTimeLabels(updatedMemorySaturationValues.timeLabels);
       setMemorySaturationDatasets(updatedMemorySaturationValues.dataset);
     }
+  };
+
+  // update metric values with the results of calling the above functions and update state
+  // BUG WITH SET INTERVAL: when you switch to a new pod, the previously selected pods' linecharts are still being rendered every 3 seconds**/
```
Comment (ntgv93) [inline src/components/K8sContainersOverview/K8sContainersOverview.js:155]: Does this bug still exist?

File: src/components/K8sContainersOverview/K8sContainersOverview.js [hunk 4]
```diff
@@ -150,6 +149,14 @@ const K8sContainersOverview = (props) => {
...
+  useEffect(async () => {
+    fetchToState();
+    // const interval = setInterval(() => fetchToState(), 3000);
+    // return () => clearInterval(interval);
```
Comment (ntgv93) [inline src/components/K8sContainersOverview/K8sContainersOverview.js:159]: Do these comments still apply?

**Comment by jackiewhitworth** [comment]
merged into pull request for jackie/merged branch

**Pull request closed without merging**